src/annotator/highlight-clusters.tsx

```diff
@@ -9,9 +9,12 @@ import type { HighlightCluster } from '../types/shared';
 import ClusterToolbar from './components/ClusterToolbar';
 import { createShadowRoot } from './util/shadow-root';
 
+import type { HighlightElement } from './highlighter';
+
 export type HighlightStyle = {
   color: string;
-  decoration: string;
+  'color-1': string;
+  'color-2': string;
```

Comment:
I will almost assuredly remove these properties from this type and derive them programmatically. It just requires some re-fiddle with the component, so I'm deferring it.